anchore_engine/clients/grype_wrapper.py

```diff
@@ -125,6 +125,7 @@ class GrypeWrapperSingleton(object):
     def __new__(cls):
         # If the singleton has not been initialized yet, do so with the instance variables below
```

Comment:
Could you add a comment here for future maintainers noting the where and why of this having to be created at bootstrap?